File: c/iniciante/1014.c
/* title: Consumo
 * number of the problem: 1014
 * date: 18/09/2020
 * author: ltsdw
 */

#include <stdio.h>

int main(void) {
    int d;
    double l;
    scanf("%d", &d);
    scanf("%lf", &l);

    printf("%.3f km/l\n", ((double) d / l));

}

File: haskell/iniciante/1014.hs
-- title: Consumo
-- number of the problem: 1014
-- date: 21/09/20
-- author: ltsdw

import Text.Printf

main :: IO ()
main = do
    d' <- getLine
    l' <- getLine

    let d = read d' :: Double
    let l = read l' :: Double

    printf "%.3f km/l\n" (d / l)

